oomph/typer.py

```diff
@@ -379,54 +375,107 @@ def _create_to_string_method(class_type: tast.Type) -> uast.FuncDef:
 
 
 class _FileTyper:
-    def __init__(self) -> None:
+    def __init__(
+        self,
+        path: pathlib.Path,
+        export_vars: List[tast.ExportVariable],
```

Comment:
```suggestion
        export_vars: List[tast.ExportVariable]
```

Then black again, does it fit on one line?